op-bindings/Makefile

```diff
@@ -1,42 +1,65 @@
 SHELL := /bin/bash
 
+pkg := bindings
```

Comment:
what is this for?

Reply:
Its defining a variable that is used as an arg to each of the scripts called

Reply:
is it a default that can be overridden?

Reply:
Its a hardcoded variable like `const foo = 'bar'` in JS, i believe that for vars referenced in `$(..)`, they can be set at runtime via cli flag but we are hardcoding the value because its generally static for us